### PR TITLE
modify typing errors

### PR DESCRIPTION
**Pull request by merong245**
I have corrected some misspelled words

### CODE REVIEW

File: docs/support.md [hunk 1]
```diff
@@ -16,4 +16,4 @@ If you want community support, or simply want to chat with friendly TypeORM enth
 
 The TypeORM core team is always ready to provide professional commercial support.
 We are ready to work with any team in any part of the world.
-[Please contact us](mailto:support@typeorm.io).
+[Please contact us](mail to:support@typeorm.io).
```
Comment (ssilve1989) [inline docs/support.md]: This is not a typo, this is so the link works to send mail.

Comment (merong245) [reply docs/support.md]: I got it. thanks

**Comment by pleerock** [comment]
Thank you very much, can you please fix conflicts issue? Thank you.

**Comment by merong245** [comment]
Checked it

**Comment by pleerock** [comment]
Thank you so much!

**Pull request merged**